Tasks/Xcode/postxcode.ts

```diff
@@ -9,6 +9,42 @@ async function run() {
     try {
         tl.setResourcePath(path.join(__dirname, 'task.json'));
 
+        //--------------------------------------------------------
+        // Test publishing - publish even if tests fail
+        //--------------------------------------------------------
+        let testResultsFiles: string;
+        const publishResults: boolean = tl.getBoolInput('publishJUnitResults', false);
+        const useXcpretty: boolean = tl.getBoolInput('useXcpretty', false);
+        const workingDir: string = tl.getPathInput('cwd');
+        
+        if (publishResults && useXcpretty) {
+            if (!useXcpretty) {
+                tl.warning(tl.loc('UseXcprettyForTestPublishing'));
+            } else {
+                testResultsFiles = tl.resolve(workingDir, '**/build/reports/junit.xml');
+
+                if (testResultsFiles && 0 !== testResultsFiles.length) {
+                    //check for pattern in testResultsFiles
+                    let matchingTestResultsFiles: string[];
+                    if (testResultsFiles.indexOf('*') >= 0 || testResultsFiles.indexOf('?') >= 0) {
```

Comment:
I realize you're just moving this code, but given that we have the path hardcoded on line 24, I think this is no longer relevant (at least the '?') since we removed xcToolReporter.